gem/spec/spec_helper.rb

```diff
@@ -13,7 +13,7 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
-require_relative '../lib/earlgrey'
+require 'earlgrey/cli'
```

Comment:
how about those:

[fastlane](https://github.com/fastlane/fastlane/search?l=Ruby&q=require_relative&type=&utf8=%E2%9C%93)
[CocoaPods](https://github.com/CocoaPods/CocoaPods/search?utf8=%E2%9C%93&q=require_relative&type=)
[Carthage](https://github.com/Carthage/Carthage/search?utf8=%E2%9C%93&q=require_relative&type=)
[Rails](https://github.com/rails/rails/search?l=Ruby&p=4&q=require_relative&type=&utf8=%E2%9C%93)
[Rake](https://github.com/ruby/rake/search?utf8=%E2%9C%93&q=require_relative&type=)
[Bundler](https://github.com/bundler/bundler/search?utf8=%E2%9C%93&q=require_relative&type=)
This list can go on and on...

None of those are either using `relative_require` or require it outside the test scope. I wouldn't want earlgrey to be the first project that does this practice. :/

Yes, it does massage your load_path, but that's based on top of knowledge of using RSpec, and you would expect RSpec would test your local lib/gem, not something else; if it does, it's a bug for RSpec.

Would you run those spec.rb directly? probably not.